ResidentInformationApi/V1/UseCase/ListContactsUseCase.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using ResidentInformationApi.V1.Boundary.Requests;
+using ResidentInformationApi.V1.Boundary.Responses;
+using ResidentInformationApi.V1.Factories;
+using ResidentInformationApi.V1.Gateways;
+
+namespace ResidentInformationApi.V1.UseCase
+{
+    public class ListContactsUseCase : IListContactsUseCase
+    {
+        private readonly IAcademyInformationGateway _academyGateway;
+        private readonly IHousingInformationGateway _housingGateway;
+        private readonly IMosaicInformationGateway _mosaicGateway;
+
+        public ListContactsUseCase(
+            IAcademyInformationGateway academyGateway,
+            IHousingInformationGateway housingGateway,
+            IMosaicInformationGateway mosaicGateway
+            )
+        {
+            _academyGateway = academyGateway;
+            _housingGateway = housingGateway;
+            _mosaicGateway = mosaicGateway;
+        }
+
+        public ContactInformationResponse Execute(ResidentQueryParam rqp)
+        {
+            var academyClaimants = _academyGateway.GetClaimantInformation(rqp);
+            var housingResidents = _housingGateway.GetResidentInformation(rqp);
+            var mosaicResidents = _mosaicGateway.GetResidentInformation(rqp);
+
+
+            return new ContactInformationResponse
+            {
+                AcademyResult = new AcademyClaimantInformationResult
+                {
+                    System = "Academy",
+                    SystemId = "123_456",
```

Comment:
What are these ID's?

Reply:
I don't actually know, I originally thought that they might be an amalgamation of person ID and claim ID, but following a discussion, it may be a more generic ID for the system itself. If it's the former, the system ID should probably be in the ClaimantInformation level.